config/val.config

```diff
@@ -0,0 +1,40 @@
+%% -*- erlang -*-
+[
+ "config/sys.config",
```

Comment:
This should really just be a testnet.config and override for sys.config not completely replacing all the below. That is, if we actually ever want to support testnet in the repo going forward. 

I don't think we should but at least the network setting in the sup and a bumped libp2p and core to pick up the crypto changes in v1.3.3 would be great